### ISSUE TITLE
Change category name in OT Collection: Creative Writing

### ISSUE THREAD

**Issue by acheveldave** [BCcampus/open-sage#221]
Hi @alex-418! I was wondering if you could change the name of a sub category in the OT collection when you get the chance.

Could you please change the Communication/Writing sub category "Creative Non-Fiction" to "Creative Writing"? Thank you!

**Comment by alex-418** [comment]
@acheveldave Sure can, this is now done. Thanks

**Comment by acheveldave** [comment]
@alex-418 Thank you!

**Comment by acheveldave** [comment]
Hi @alex-418, I've noticed that Creative Writing appears to be listed twice in the sub disciplines list once you've selected Communication/Writing as a discipline when adding a resource in SOLR. Then, once you've selected Creative Writing as a sub discipline and save, Creative Writing appears twice in the selected sub disciplines. I've attached two screenshots for reference.


![creativewriting](https://user-images.githubusercontent.com/59974948/73018362-59f8e180-3dd6-11ea-8a8c-9489d223082a.PNG)

![creativewriting2](https://user-images.githubusercontent.com/59974948/73018393-65e4a380-3dd6-11ea-9138-e2f91a523a46.PNG)

**Comment by alex-418** [comment]
Hi @acheveldave I've confirmed there's no duplicate in the collection definitions, and I've tested this on my end to try and replicate your results. But mine works as expected, without the duplicate. I suspect this issue to be browser related. What happens when you try in a different browser, and also what happens once you've saved it? Please try and let me know, thank you.

**Comment by acheveldave** [comment]
Hi @alex-418, I originally had the problem in Firefox, and when I tried it in Google Chrome today, I saw that I am still seeing two versions of "Creative Writing" in the list. When I select one Creative Writing and save, both Creative Writings are in the selected sub disciplines box on the right. When I try removing one of the Creative Writings and save, both are still in the selected sub disciplines box.

![creatwrit](https://user-images.githubusercontent.com/59974948/73202107-380da080-40ef-11ea-8d5e-b5b61983e90d.PNG)

Both books in the Creative Writing sub discipline appear normally in the collection, although in SOLR, each has their discipline is listed as "Creative Writing, Creative Writing," when it should be "Communication/Writing, Creative Writing."
